### DISCUSSION TITLE
PiMox HAOS VM -> More Diskspace ?

### DISCUSSION THREAD

**Discussion by MagicMikeMirror** [tteck/Proxmox#3916] [Help Needed] [answered]
Hi Guys,
sorry for this maybe dumb question, i'm totally new to HA and Proxmox and so on .. 
Is there a possibility to get more Diskspace while using this great Script:
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/vm/pimox-haos-vm.sh)"

Thank you in advance

**Comment by tteck** [comment, marked as answer]
resize disk

![image](https://github.com/user-attachments/assets/f4b05630-c500-40a2-84c2-7f901084b241)

**Reply by MagicMikeMirror** [reply]
O M F G .. what a stupid noob failure from me .. thank you so much man !! 
Not all Heroes wear Capes !
